state-sync/src/coordinator.rs

```diff
@@ -821,13 +826,12 @@ impl<T: ExecutorProxyTrait> StateSyncCoordinator<T> {
             let end_of_epoch_li = self
                 .executor_proxy
                 .get_epoch_change_ledger_info(request.current_epoch)?;
-            ensure!(
-                end_of_epoch_li.ledger_info().version() >= request.known_version,
-                "waypoint request's current_epoch (epoch {}, version {}) < waypoint request's known_version {}",
-                end_of_epoch_li.ledger_info().epoch(),
-                end_of_epoch_li.ledger_info().version(),
-                request.known_version,
-            );
+            if end_of_epoch_li.ledger_info().version() < request.known_version {
+                return Err(Error::UnexpectedError(format!(                "Waypoint request's current_epoch (epoch {}, version {}) < waypoint request's known_version {}",
+                                                                          end_of_epoch_li.ledger_info().epoch(),
+                                                                          end_of_epoch_li.ledger_info().version(),
+                                                                          request.known_version,)));
```

Comment:
This spacing is way off, something went weird

Reply:
Thanks Greg! Weird that the auto formatter didn't pick that up... 🤔

Reply:
Auto formatter doesn't always pickup formatting inside macros, because in "theory" it could be important to the macro